lib/ratfun.gi

```diff
@@ -1350,6 +1350,20 @@ function(f,g)
   TryNextMethod();
 end);
 
+#############################################################################
```

Comment:
This change must also be applied to `hpcgap/lib/ratfun.gi`

Reply:
Thanks! I wasn't aware that hpcgap has its own lib. I'll add it.